src/test/java/com/nerodesk/om/aws/AwsDocsTest.java

```diff
@@ -40,6 +48,34 @@
  * @version $Id$
  */
 public final class AwsDocsTest {
+
+    /**
+     * AwsDoc can return the size of its contents.
+     * @throws Exception If something goes wrong.
+     * @todo #131:30min Ideally we should be using mock classes from jcabi-s3 in
+     *  this test. However, the ObjectMetadata that is returned by the mocking
+     *  framework does not show an accurate contentLength. This is the reason
+     *  we're using Mockito. When jcabi-s3 supports this feature (see
+     *  https://github.com/jcabi/jcabi-s3/issues/25), let's migrate this test
+     *  to com.jcabi.s3.mock classes.
+     */
+    @Test
+    public void fetchesSize() throws Exception {
+        final long size = Tv.THOUSAND;
+        final Bucket bucket = Mockito.mock(Bucket.class);
+        final Ocket ocket = Mockito.mock(Ocket.class);
```

Comment:
We can't (with current jcabi-s3 version). See the puzzle that I created, issue has been raised in jcabi-s3.